lsix

```diff
@@ -65,9 +65,7 @@ if command -v gsed >/dev/null; then
     alias sed=gsed		# Use GNU sed for MacOS & BSD.
 fi
 
-if [[ "$COMSPEC" ]]; then
-    alias convert="magick convert" # Shun MS Windows' "convert" command.
-fi
+alias convert="magick" # Shun MS Windows' "convert" command. Also fix some silly system aliases.
```

Comment:
A good idea, but it would break ImageMagick 6 which did not have a `magick` command.